api/api_storage.go

```diff
@@ -5,80 +5,12 @@ import (
 
 	"github.com/filecoin-project/go-address"
 	"github.com/filecoin-project/go-sectorbuilder"
+	s2 "github.com/filecoin-project/go-storage-miner"
```

Comment:
Why s2? Can this just be 'sealing'?

Reply:
We can alias to whatever you like. The package name, by default, is `storage` (`sealing` and `storage` are collapsed in go-storage-miner).